internal/collector/nginxossreceiver/metadata.yaml

```diff
@@ -7,7 +7,21 @@ status:
     beta: [metrics]
   distributions: [contrib]
   codeowners:
-    active: [olli-holmala, oliveromahony, apgralG, dhurley, craigell]
+    active: [olli-holmala, oliveromahony, apgralG, dhurley, craigell, sean-breen]
+
+resource_attributes:
+  resource.id:
+    description: The host machine id.
+    type: string
+    enabled: true
+  nginx.instance.id:
```

Comment:
Should `nginx.instance.id` and `nginx.instance.type` be renamed to `instance.id` and `instance.type`?